src/Components/Dashboard/OpportunityPannel/OpportunityCreation.jsx

```diff
@@ -0,0 +1,283 @@
+						className="input-style"
+					/>
+					{errors.opportunityRole && (
+						<span className="error-style">Opportunity Role is required</span>
+					)}
+				</div>
+				{/*--------------Opportunity Tags------------*/}
+				<ChipInput
+					label="Opportunity Tags"
+					id="opportunityTags"
+					placeholder="e.g., B.Tech & press Enter"
+					register={register}
+					errors={errors}
+					setValue={setValue}
+					getValues={getValues}
+				/>
+				<div className="flex flex-col md:flex-row w-full gap-3">
+					{/* -------------OPPORTUNITY LOCATION------------- */}
+					<div className="flex flex-col space-y-2 md:w-1/2 w-full">
+						<label className="label-style" htmlFor="opportunityLocation">
+							Opportunity Location <sup className="text-pink-200">*</sup>
+						</label>
+						<input
+							id="opportunityLocation"
+							placeholder="e.g., Bangalore"
+							{...register("opportunityLocation", { required: true })}
+							className="input-style"
+						/>
+						{errors.opportunityLocation && (
+							<span className="error-style">Opportunity Role is required</span>
+						)}
+					</div>
+					{/* -------------OPPORTUNITY PACKAGE------------- */}
+					<div className="flex flex-col space-y-2 md:w-1/2 w-full">
+						<label className="label-style" htmlFor="opportunityPackage">
+							Opportunity Package <sup className="text-pink-200">*</sup>
+						</label>
+						<div className="relative">
+							<input
+								id="opportunityPackage"
+								placeholder="e.g., 2500000"
+								{...register("opportunityPackage", {
+									required: "Opportunity Package is required",
+									pattern: {
+										value: /^(0\.0*[1-9]\d*|[1-9]\d*(\.\d+)?)$/,
+										message:
+											"Opportunity Package must be a positive number greater than zero",
+									},
+								})}
+								className="!pl-12 input-style"
+							/>
+							{errors.opportunityPackage && (
+								<span className="error-style">
+									{errors.opportunityPackage.message}
+								</span>
+							)}
+							<HiOutlineCurrencyRupee className="absolute left-3 top-1/2 inline-block -translate-y-1/2 text-2xl text-richblack-800 dark:text-richblack-200" />
+						</div>
+					</div>
+				</div>
+				{/*---------------ELIGIBILITY REQUIREMENTS------------ */}
+				<RequirementField
+					id="eligibilityCriteria"
+					label="Eligibility Criteria"
+					register={register}
+					setValue={setValue}
+					errors={errors}
+					getValues={getValues}
+					placeholder="Add Eligibility Criteria"
+				/>
+				<div className="flex lg:justify-end justify-center gap-x-2 flex-wrap gap-y-2">
+					{editOpportunity && (
+						<button
+							onClick={() => {
+								dispatch(setEditOpportunity(false), dispatch(setStep(2)));
+							}}
+							disabled={loading}
+							className={`flex w-[80%] lg:w-fit cursor-pointer items-center justify-center gap-x-2 rounded-md bg-richblack-700 hover:bg-richblack-900 py-2 px-5 text-base font-semibold text-gray-100`}
```

Comment:
Ig the button will look good if change the text to `text-sm`
![Screenshot from 2024-04-11 10-42-24](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/fc15d04c-cbe3-4405-b0eb-a104d62d62fa)

here is the screenshot after changes applied that I suggested
![image](https://github.com/Opportunity-Nexus/Opportunity-Nexus/assets/83964826/68e60903-682b-4f3e-9bd4-3364ff96a9cb)